### ISSUE TITLE
xml2csv does not generate csv output file in python 3.9

### ISSUE THREAD

**Issue by arghyas73** [eclipse/sumo#8894]
I am using SUMO 1.9.2 on Windows 10 and Python 3.9 installed. I noticed that xml2csv does not produce the output csv file. However the same code is able to generate the csv file on another laptop in which I have Python 3.8 on Ubuntu.

**Comment by namdre** [comment]
please try to include the python interpreter in your command:
`c:/path/to/python c:/path/to/sumo/tools/xml/xml2csv.py  INPUT_FILE`
instead of
`c:/path/to/sumo/tools/xml/xml2csv.py  INPUT_FILE`

reason: sometimes windows does not pass on arguments due to a configuration problem"

**Comment by arghyas73** [comment]
Thanks for the reply. I tried to include C:/Users/hp/AppData/Local/Programs/Python/Python39 in the command. But it is still not producing the csv file.  On the other hand, I notice that xml2csv tool in SUMO uses optparse., which is deprecated since python version 3.2.  Though it has been working in python 3.8, could this issue be related to change in python support to optparse?

**Comment by namdre** [comment]
The python 3.9 documentation suggests that optparse is still supported (though deprecated). 
More things to try on your end:
- are other sumo python tools working as expected (i.e. randomTrips.py)?
- are other input files for xml2csv working on the windows machine?
- run `python -m trace --trace xml2csv.py input.xml > log.txt` and attach the zipped log file

**Comment by arghyas73** [comment]
Now I have found the reason for the code not working. This is because of the white space that the file path contains, in case of windows, Since I ran the code on Windows after I had upgraded to Python 3.9, I thought it was a Python 3.9 issue. Now haveI changed the code like this:
`os.system('python'+' '+ '"'+xml2csvPath+'"'+' '+'path/to/xmlfile.xml')`
and it works (I understand that subprocess can also be used). Other sumo python tools are also working.

Sorry for my oversight, and thanks a lot @namdre !
